src/pages/HomePage.js

```diff
@@ -19,25 +19,34 @@ const areasTablet = `
 `
 
 const HomePage = () => (
+  <div>
+    <GlobalStyle bgColor="#FFF6E5" />
     <Composition
-        areas={areasMobile}
-        areasMd={areasTablet}
-        padding={2}
-        templateColsMd="40% auto"
-        gap={2}
+      areas={areasMobile}
+      areasMd={areasTablet}
+      padding={2}
+      templateColsMd="40% auto"
```

Comment:
You can also combine `areas` and `templateCols` using the `template` prop:

```jsx
const areasTablet = `
  info image
  / 40% auto
`

<Composition
  areas={areasMobile}
  templateMd={areasTablet} // note the prop "template"
>
```

Reply:
I can't
it breaks this part